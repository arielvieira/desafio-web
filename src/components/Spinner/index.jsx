import React from 'react';

const LoadingPage = () => (
  <div className="spinner">
    <img className="spinner__image" src="/static/img/loader.gif" alt="Loader gif" />
  </div>
);

export default LoadingPage;
