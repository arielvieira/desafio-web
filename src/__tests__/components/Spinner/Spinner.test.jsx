import React from 'react';
import { shallow } from 'enzyme';

import Spinner from '../../../components/Spinner';

describe('Spinner', () => {
  it('should render Spinner correctly', () => {
    const wrapper = shallow(
      <Spinner />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
