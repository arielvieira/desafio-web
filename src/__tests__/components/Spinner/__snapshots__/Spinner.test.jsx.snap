// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render Spinner correctly 1`] = `
<div
  className="spinner"
>
  <img
    alt="Loader gif"
    className="spinner__image"
    src="/static/img/loader.gif"
  />
</div>
`;
